Documentation/user-guides/running-exporters.md

```diff
@@ -0,0 +1,68 @@
+# Running Exporters
+Running exporters and scraping them with Prometheus configured by the prometheus-operator.
+
+## The goal of ServiceMonitor(s)
+The goal for one ServiceMonitor should be to cover a large number of `Service`s.
+This can be achieved by creating a generic `ServiceMonitor` that for example would .
```

Comment:
Incomplete sentence, it would make sense if we just cut ~~that for example would~~.